pkg/apis/pingcap/v1alpha1/tidbclusterautoscaler_types.go

```diff
@@ -112,6 +112,24 @@ type BasicAutoScalerSpec struct {
 	// If not set, the default metric will be set to 80% average CPU utilization.
 	// +optional
 	Metrics []v2beta2.MetricSpec `json:"metrics,omitempty"`
+
+	// MetricsTimeDuration describe the Time duration to be queried in the Prometheus
+	// +optional
+	MetricsTimeDuration *string `json:"MetricsTimeDuration,omitempty"`
+
+	// ScaleOutThreshold describe the consecutive threshold for the auto-scaling,
+	// if the consecutive counts of the scale-out result in auto-scaling reach this number,
+	// the auto-scaling would be performed.
+	// If not set, the default value is 3.
+	// +optional
+	ScaleOutThreshold *int32 `json:"scaleOutThreshold,omitempty"`
+
+	// ScaleInThreshold describe the consecutive threshold for the auto-scaling,
+	// if the consecutive counts of the scale-int result in auto-scaling reach this number,
+	// the auto-scaling would be performed.
+	// If not set, the default value is 3.
```

Comment:
We should be more conservative for scaling in, default to a bigger value, e.g. 5?

Reply:
There is no necessary that the default value of scale-in threshold should be bigger than scale-out's. 3 is big enough for both of them.

Reply:
It depends, check the third comment of https://github.com/pingcap/tidb-operator/pull/1703/files?file-filters%5B%5D=.go#r379976699, is it big enough?

Reply:
updated.